basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/jmsbridge/JMSBridgeAttributesTestCase.java

```diff
@@ -0,0 +1,498 @@
+        }
+    }
+
+    @Test
+    public void editSourceUser() throws Exception {
+        final String VALUE = RandomStringUtils.randomAlphanumeric(5);
+
+        ModelNodeResult defaultValueResult = operations.readAttribute(JMS_BRIDGE_ADDRESS, SOURCE_USER_IDENTIFIER);
+        defaultValueResult.assertSuccess();
+        ModelNode defaultValue = defaultValueResult.value();
+        try {
+            new ConfigChecker.Builder(client, JMS_BRIDGE_ADDRESS)
+                    .configFragment(page.getConfigFragment())
+                    .editAndSave(ConfigChecker.InputType.TEXT, SOURCE_USER_IDENTIFIER, VALUE)
+                    .verifyFormSaved()
+                    .verifyAttribute(SOURCE_USER_IDENTIFIER, VALUE);
+        } finally {
+            operations.writeAttribute(JMS_BRIDGE_ADDRESS, SOURCE_USER_IDENTIFIER, defaultValue);
+        }
+    }
+
+    @Test
+    public void editTargetConnectionFactory() throws Exception {
+        final String VALUE = RandomStringUtils.randomAlphanumeric(5);
+
+        ModelNodeResult defaultValueResult = operations.readAttribute(JMS_BRIDGE_ADDRESS, TARGET_CONNECTION_FACTORY_IDENTIFIER);
+        defaultValueResult.assertSuccess();
+        ModelNode defaultValue = defaultValueResult.value();
+        try {
+            new ConfigChecker.Builder(client, JMS_BRIDGE_ADDRESS)
+                    .configFragment(page.getConfigFragment())
+                    .editAndSave(ConfigChecker.InputType.TEXT, TARGET_CONNECTION_FACTORY_IDENTIFIER, VALUE)
+                    .verifyFormSaved()
+                    .verifyAttribute(TARGET_CONNECTION_FACTORY_IDENTIFIER, VALUE);
+        } finally {
+            operations.writeAttribute(JMS_BRIDGE_ADDRESS, TARGET_CONNECTION_FACTORY_IDENTIFIER, defaultValue);
+        }
+    }
+
+    @Test
+    public void editTargetDestination() throws Exception {
+        final String VALUE = RandomStringUtils.randomAlphanumeric(5);
+
+        ModelNodeResult defaultValueResult = operations.readAttribute(JMS_BRIDGE_ADDRESS, TARGET_DESTINATION_IDENTIFIER);
+        defaultValueResult.assertSuccess();
+        ModelNode defaultValue = defaultValueResult.value();
+        try {
+            new ConfigChecker.Builder(client, JMS_BRIDGE_ADDRESS)
+                    .configFragment(page.getConfigFragment())
+                    .editAndSave(ConfigChecker.InputType.TEXT, TARGET_DESTINATION_IDENTIFIER, VALUE)
+                    .verifyFormSaved()
+                    .verifyAttribute(TARGET_DESTINATION_IDENTIFIER, VALUE);
+        } finally {
+            operations.writeAttribute(JMS_BRIDGE_ADDRESS, TARGET_DESTINATION_IDENTIFIER, defaultValue);
+        }
+    }
+
+    @Test
+    public void editTargetPassword() throws Exception {
+        final String VALUE = RandomStringUtils.randomAlphanumeric(5);
+
+        ModelNodeResult defaultValueResult = operations.readAttribute(JMS_BRIDGE_ADDRESS, TARGET_PASSWORD_IDENTIFIER);
```

Comment:
When backuping value, it is good to read it ignoring defaults => with `ReadAttributeOption.NOT_INCLUDE_DEFAULTS`.
=> default value and undefined value are not exactly the same values :-)